EasyCommands/BlockHandlers/GyroscopeBlockHandlers.cs

```diff
@@ -29,19 +29,19 @@ public class GyroscopeBlockHandler<T> : FunctionalBlockHandler<T> where T : clas
 
                 var overrideHandler = DirectionalTypedHandler(Direction.NONE,
                         TypeHandler(ReturnTypedHandler(Return.VECTOR,
-                            TypeHandler(VectorHandler(b => new Vector3D(GetPitch(b), GetYaw(b), GetRoll(b)), (b, v) => {
-                                SetPitch(b, (float)v.X);
-                                SetYaw(b, (float)v.Y);
-                                SetRoll(b, (float)v.Z);
+                            TypeHandler(VectorHandler(b => new Vector3D(b.Pitch * RadiansPerSecToRPM, b.Yaw * RadiansPerSecToRPM, b.Roll * RadiansPerSecToRPM), (b, v) => {
```

Comment:
nit: I'm pretty sure you can pull out the multiplication:
```RadiansPerSecToRPM * new Vector3D(b.Pitch, b.Yaw, b.Roll)```

Reply:
that is indeed possible but does lead to problems with the test since the numbers are not truncated to float again and so do not match when the multiplication is outside.

Reply:
Since i cannot find the comment with accuraccy again i will reply here . with the current order of operation the output should be exactly the same as before.

But this is a general problem with vectors being double and numbers being float internally.

Reply:
Indeed, I've considered changing the internals to use doubles.  I only did float because most properties in SE were float so it was less chars, lol.